packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -3964,14 +4032,14 @@ async function waitForPublishedEvents<E extends NetworkControllerEvents>(
         }
       };
 
-      const stopTimer = () => {
+      const removeAllListenersTimer = () => {
```

Comment:
This change is unclear to me. The timer doesn't remove all listeners; it just force-ends the test. Maybe this was a find-and-replace sort of thing?

Reply:
oh yea search and replace rekd!